src/hci/commands/ifmgmt_cmd.c

```diff
@@ -202,6 +204,9 @@ static struct option_descriptor ifconf_opts[] = {
 	OPTION_DESC ( "configurator", 'c', required_argument,
 		      struct ifconf_options, configurator,
 		      parse_netdev_configurator ),
+	OPTION_DESC ( "timeout", 't', required_argument,
```

Comment:
I'm tempted to use `timeout_ms` to explicit the unit.

Reply:
I chose `timeout` to stay consistent with the rest of the user commands:
```
$ rg \"timeout\"
src/image/script.c
375:	OPTION_DESC ( "timeout", 't', required_argument,

src/hci/commands/sync_cmd.c
47:	OPTION_DESC ( "timeout", 't', required_argument,

src/hci/commands/ping_cmd.c
65:	OPTION_DESC ( "timeout", 't', required_argument,

src/hci/commands/menu_cmd.c
213:	OPTION_DESC ( "timeout", 't', required_argument,

src/hci/commands/image_trust_cmd.c
103:	OPTION_DESC ( "timeout", 't', required_argument,

src/hci/commands/image_cmd.c
67:		OPTION_DESC ( "timeout", 't', required_argument,```